src/logic/contracts/generateBatchRequests.ts

```diff
@@ -14,25 +14,33 @@ import { AbiItem } from 'web3-utils'
  */
 type MethodsArgsType = Array<string | number>
 
- interface Props {
+interface Props {
   abi: AbiItem[]
   address: string
   batch?: BatchRequest
   context?: unknown
-  methods: Array<string | {method: string, type?: string, args: MethodsArgsType }>
- }
+  methods: Array<string | { method: string; type?: string; args: MethodsArgsType }>
```

Comment:
Why shouldn't we use the simple coma here `, ?`

Reply:
it was prettier who changed it